frontend/msLoader.pas

```diff
@@ -106,6 +106,7 @@ function InitBase: boolean;
   wbEditAllowed := True;
   wbContainerHandler := wbCreateContainerHandler;
   wbContainerHandler._AddRef;
+  wbLocalizationHandler.NoTranslate := true;
```

Comment:
I don't think you can do this, because when you copy a localized string in xEdit it copies the localized string value.  A localized string is basically just a `uint32` (`cardinal` in Delphi-land) which points to a specific id in the `strings` file associated with the ESP being processed.  For that reason, when copying a localized string it makes sense to copy not the `id` but the localized string value itself, because the `id` isn't meaningful outside of whatever plugin the record is getting copied from.

But I could also be misinterpreting what this option does?  Mostly saying this based on your commit comment.

Reply:
Yeah you are right. That setting does not do what I thought I did. It makes the errors go away but the strings are still wrong.